P5/Source/Guidelines/en/CMC-ComputerMediatedCommunication.xml

```diff
@@ -0,0 +1,1302 @@
+        example; this phenomenon is not in any way unique to X (Twitter), however.</p>
+      <p>
+        <egXML xmlns="http://www.tei-c.org/ns/Examples" xml:id="ex.tweets" xml:lang="de">
+          <post modality="written" generatedBy="human" type="tweet" who="#u1"
+            xml:id="cmc_post_1043796550101716993" synch="#tweetsbcrn18.t004" xml:lang="de">
+            <ptr type="retweet" target="#cmc_post_1043796093786566656"/> Ich mich auch? <ref
+              type="hashtag" target="https://twitter.com/hashtag/dynamicduo?src=hash"
+              >#dynamicduo</ref>
+            <ref type="hashtag"
+              target="https://twitter.com/hashtag/wirk%C3%BCmmernunsauchumIhrenEmpfang?src=hash"
+              >#wirkümmernunsauchumIhrenEmpfang</ref>
+            <ref type="hashtag" target="https://twitter.com/hashtag/bcrn18?src=hash">#bcrn18</ref>
+            <ref type="hashtag" target="https://twitter.com/hashtag/wikidach?src=hash"
+              >#wikidach</ref>
+          </post>
+          <post modality="written" generatedBy="human" type="tweet" who="#u2"
+            synch="#tweetsbcrn18.t003" xml:lang="de" xml:id="cmc_post_1043796093786566656">
+            <time generatedBy="system"> 14:35 </time> Immer wieder gerne. Kann ich mich schon für
+            nächstes Jahr als Empfangs- <ref type="hashtag"
+              target="https://twitter.com/hashtag/Engel?src=hash">#Engel</ref> für das nächste
+            BarCamp bewerben <w pos="EMO">&#x1F92A;</w>
+            <ref type="hashtag" target="https://twitter.com/hashtag/bcrn18?src=hash">#bcrn18</ref>
+            <trailer>
+              <fs>
+                <f name="favoritecount">
+                  <numeric value="4"/>
+                </f>
+              </fs>
+            </trailer>
+          </post>
+        </egXML> In the preceding example, we first encode the type of post (in this case, a tweet)
+        in the <att>type</att> attribute, and if it is a specific subtype of tweet, that will be
+        encoded in a <att>subtype</att> attribute of the <gi>post</gi> element accordingly.</p>
+      <p>Second, the original tweet-ID as supplied by X (Twitter) is contained in the value of the
+        global attribute <att>xml:id</att>.</p>
+      <p>Third, a retweet and its corresponding retweeted tweet are encoded as two separate posts
+        each with its own set of attributes. The post representing the retweet itself does not
+        contain or duplicate the content of the retweeted tweet. Instead it refers to the ID of the
+        retweeted tweet via a <gi>ptr</gi> in the post content. All original
+        content of the retweet goes in the content of the <gi>post</gi> element as well.</p>
+      <p>Fourth, hashtags occur in the body of tweets and are links like any other hyperlink and
+        should be encoded using the element <gi>ref</gi>. The <att>type</att> attribute of the ref
```

Comment:
ref should be <gi>ref</gi>

Reply:
Fixed in previous re-write of this paragraph at https://github.com/TEIC/TEI/commit/31f496f6053bb06fd5fc519875505f73d30f6d8e.